Events/2022-03-diverse-leung.md

```diff
@@ -0,0 +1,45 @@
+# DiveRSE Speaker Series: Mary Leung
+
+- Date: 2022-03-22
+- Location: Online
+- Event Website: https://diverse-rse.github.io/events/2022-03-22
+- Organizers: DiveRSE & Society of Research Software Engineering 
+			   
+[DiveRSE (Diverse RSE)](https://diverse-rse.github.io/) is a series of talks and related activities including
+discussion and panel sessions designed to support and raise
+awareness of EDI challenges and successes within the research
+software community. The series runs from March 2022 and is linked to
+the [“Vive la différence - research software engineers” workshop](https://www.researchsoft.org/events/2022-04/)
+being hosted at the Lorentz Centre in The Netherlands in April 2022.
```

Comment:
A couple of questions here
1) Is *this* talk associated with the workshop?
2) Is the DiveRSE series expected to continue beyond the workshop?

If DiveRSE expects to have its own identity and run longer, I'd suggest dropping this sentence here and instead putting a note before or after the abstract linking this talk and the workshop.  Other DiveRSE talks related to the workshop could do the same.  But after the workshop-related programming is over, you would drop that.